code/modules/codex/entries/atmospherics.dm

```diff
@@ -0,0 +1,212 @@
+/* all of these have new/different typepaths. come back when people care about atmos.
```

Comment:
It's fine to use some commented code when porting a PR, but the entire file should not be just commented out code.

Reply:
This was something I had fixed in the typepath changes, but got lost during the revert. I didn't feel like redoing my work at the time.

Reply:
every path was changed in Spooky's TG pipes update.